client/my-sites/pages/page/index.js

```diff
@@ -203,6 +205,31 @@ class Page extends Component {
 		);
 	}
 
+	setFrontPage() {
```

Comment:
This works, because the `alert` statement is still executed, but this shouldn't have `PopoverMenuItem` or `onClick` (you already did those parts during `getFrontPageItem`).